common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/LiferayCore.java

```diff
@@ -295,4 +366,27 @@ public void stop( BundleContext context ) throws Exception
         }
     }
 
-}
+    private boolean validateSDK( final String loc)
```

Comment:
Don't we already have a validate SDK method?  It doesn't seem like we need another one of these.